recipe.cake

```diff
@@ -54,14 +56,35 @@ Func<List<ILMergeConfig>> getILMergeConfigs = () =>
         Output = BuildParameters.Paths.Directories.PublishedLibraries + "/chocolatey_merged/chocolatey.dll",
         PrimaryAssemblyName = BuildParameters.Paths.Directories.PublishedLibraries + "/chocolatey/chocolatey.dll",
         AssemblyPaths = assembliesToILMerge });
+    
+    // var powershellAssembliesToMerge = GetFiles(BuildParameters.Paths.Directories.PublishedLibraries + "/Chocolatey.PowerShell/netstandard2.0/*.dll")
+    //                                 - GetFiles(BuildParameters.Paths.Directories.PublishedLibraries + "/Chocolatey.PowerShell/netstandard2.0/Chocolatey.PowerShell.dll")
+    //                                 - GetFiles(BuildParameters.Paths.Directories.PublishedLibraries + "/Chocolatey.PowerShell/netstandard2.0/System.Management.Automation.dll");
+
+    // Information("The following assemblies have been selected to be ILMerged for Chocolatey.PowerShell.dll...");
+    // foreach (var assemblyToILMerge in powershellAssembliesToMerge)
+    // {
+    //     Information(assemblyToILMerge.FullPath);
+    // }
+
+    mergeConfigs.Add(new ILMergeConfig() {
+        KeyFile = BuildParameters.StrongNameKeyPath,
+        LogFile = BuildParameters.Paths.Directories.Build + "/ilmerge-chocolateypowershelldll.log",
+        TargetPlatform = targetPlatform,
+        Target = "dll",
+        Internalize = BuildParameters.RootDirectoryPath + "/src/chocolatey/ilmerge.internalize.ignore.dll.txt",
+        Output = BuildParameters.Paths.Directories.PublishedLibraries + "/Chocolatey.PowerShell_signed/Chocolatey.PowerShell.dll",

```

Comment:
we discussed this one, we're not actually merging anything, but for whatever reason the `filesToSign` was not enough on its own to get the assembly to be signed.

Reply:
Revisited this just now and I _think_ this is unnecessary, just testing and will remove this if we don't need it.

Looks like adding the ref to the SolutionVersion.cs that we talked about last week or so would've fixed the signing issue as well, but we hadn't noticed that at the time, since the ILMerging _also_ fixed it (I assume because it explicitly specifies the key file in the ILMerge config.

Reply:
Yep, this can be removed, we don't need to call ILMerge for it at all.